src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,24 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator<C extends Candidate> implements Predicate<C> {
```

Comment:
```suggestion
public class CandidateValidator implements Predicate<Candidate> {
```
Don’t  parametrize your class